exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/AzureClientConfig.java

```diff
@@ -0,0 +1,4 @@
+package com.netflix.exhibitor.core.azure;
+
+public class AzureClientConfig {
```

Comment:
please remove if unused - if it's a 'marker' class (thinks Serializable) consider making it an interface instead.